common/src/types/chain_id.rs

```diff
@@ -34,9 +36,12 @@ pub enum ChainId {
     /// Starknet mainnet
     #[serde(rename = "mainnet")]
     Mainnet,
-    /// Devnet at localhost:5050
+    /// A locally hosted devnet node
     #[serde(rename = "devnet")]
     Devnet,
+    /// A Katana devnet node at `localhost:5050`
+    #[serde(rename = "katana")]
+    Katana,
```

Comment:
do we need both of these options?

Reply:
We're still using the `devnet-sequencer` elsewhere (i.e. for testing our FE) so we will support it for the time being